### ISSUE TITLE
fixed position of  submit_line in change_form

### ISSUE THREAD

**Issue by nordbit** [django-admin-bootstrapped/django-admin-bootstrapped#212]
Sometimes forms are long, everytime when i must save, even after small correct at the top of form i need scroll to bottom of the page..
We can add class .fixed-bottom-right to form-actions with css:
```
postition:fixed;
bottom:10px;
right:10px;
````
Another solution is add extra block in navbar with icon-only form after logo for form-action (common used in Android application)

**Comment by xrmx** [comment]
Thanks for the suggestion. I'll give it a try, hopefully next week.

**Comment by xrmx** [comment]
I think having form-actions behave like the navbar where it's hamburger menu on small screens and normal menu on biggers could be interesting.

**Comment by olivierdalang** [comment]
Hi,
Not sure why, but here this seems implemented (even if I installed using pip). However, it doesn't display nicely on small screens and/or long forms, since the actions buttons float over the last fields (see screenshot).
I like it more with buttons at the bottom...
Thanks !

![bug_css_django](https://cloud.githubusercontent.com/assets/1894106/9964884/576231c6-5e22-11e5-8235-847ac0118b6b.png)

**Comment by xrmx** [comment]
@olivierdalang i never test it under 1024px wide screens, i think that's really too painful to use the django admin interface in such a small screen.

**Comment by olivierdalang** [comment]
Heh i thought that was the idea of responsive layouts!!  By the way it displays very well besides of this particular problem...

**Comment by bformet** [comment]
I agree the overlap on small screens needs to be fixed, but I definitely do like the fixed position of the submit_line. Very handy on long change forms.
I'd love to see it come back (maybe as a setting?)

I can imagine two workarounds:
1) increase the body's padding-bottom
2) disable the fixed position on small screens via a media query

**Comment by securedirective** [comment]
Strangely, this was already implemented in v2.5.5

Though by looking at the repo code history, I can't figure out which commit removed it, if I install v2.5.5 manually I get the nice floating submit buttons.

    pip install django-admin-bootstrapped==2.5.5

...overrides.css look like this:

~~~
/* overrides to default admin and bootstrap styles */

.fixed-save {
    position: fixed;
    bottom: 10px;
    right: 10px;
}

body {
...
~~~

...and change_form.html looks like this:

~~~
...
    <div class="form-actions fixed-save">
        {% block submit_buttons_bottom %}{% submit_row %}{% endblock %}
    </div>
...
~~~

I like the floating submit buttons, since my form is very long. But ever since v2.5.6, the reference to `fixed-save` has been removed. Will it be coming back?

**Comment by xrmx** [comment]
@securedirective override.css hasn't been touched since july which is before 2.5.5 was released. Can you dig a bit more to find out the commit that removed that?

**Comment by xrmx** [comment]
@bformet pull requests are indeed very welcome :)

**Comment by securedirective** [comment]
@xrmx Okay, I've searched through your repo on github and cannot find any reference to the `.fixed-save` css block. I used both github's repository search and `git log -S 'fixed-save'`. I then compared your release versions to the appropriate commits:

* Release 2.5.4, installed using pip, matches perfectly with your repo as of commit 1668ec82de "Bump version to 2.5.4".
* Release 2.5.6, installed using pip, matches perfectly with your repo as of commit f536bfadae "Bump version to 2.5.6".
* But Release 2.5.5, installed using pip, **does not match** your repo as of commit e71c29967d "Bump release to 2.5.5". The only differences are the ones I pointed out. The code installed by running `pip install django-admin-bootstrapped==2.5.5` has these additional changes that are not from your repo...

`/static/admin/css/overrides.css` has an additional CSS block at the very beginning:
~~~
.fixed-save {
    position: fixed;
    bottom: 10px;
    right: 10px;
}
~~~

`/templates/admin/change_form.html` has an extra "fixed-saved" class added to the last DIV:
~~~
<div class="form-actions fixed-save">
~~~

Any way we could get these same 2 changes back for version 2.5.7 ? It was nice, having floating submit buttons.

**Comment by xrmx** [comment]
Damn, that means i was so dumb to package some local modified files instead of the tagged stuff.
